src/uu/cut/src/cut.rs

```diff
@@ -449,9 +556,13 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
                 };
 
                 let only_delimited = matches.get_flag(options::ONLY_DELIMITED);
+                let whitespace_delimited = matches.get_flag(options::WHITESPACE_DELIMITED);
                 let zero_terminated = matches.get_flag(options::ZERO_TERMINATED);
 
                 match matches.get_one::<String>(options::DELIMITER).map(|s| s.as_str()) {
+                    Some(_) if whitespace_delimited => {
+                            Err("invalid input: Only one of --delimiter (-d) or -w option can be specified".into())
+                        }
```

Comment:
What is FreeBSD's behaviour here? Usually options in coreutils just override each other.

Reply:
FreeBSD allows only one of `-w` or `-d` options, not both at the same time.

```sh
$ /usr/bin/cut -d' ' -w -f1 input.tsv
usage: cut -b list [-n] [file ...]
       cut -c list [file ...]
       cut -f list [-s] [-w | -d delim] [file ...]
```

https://github.com/freebsd/freebsd-src/blob/373ffc62c158e52cde86a5b934ab4a51307f9f2e/usr.bin/cut/cut.c#L129

Reply:
I guess that's in line with the behaviour for the other arguments to `cut` as well. Thanks for checking!